packages/components/src/horizontal-bar-list/stats-card.tsx

```diff
@@ -29,7 +29,7 @@ const StatsCard = ( {
 			{ title }
 		</a>
 	) : (
-		<div className={ `${ BASE_CLASS_NAME }-header__title` }>{ title }</div>
+		<h3 className={ `${ BASE_CLASS_NAME }-header__title` }>{ title }</h3>
```

Comment:
Setting a header hierarchy in a generalized `StatsCard` might lead to unintended consequences. For example, how does this impact accessibility of the WordAds Stats dashboard?

Maybe we can have a configurable `titleAriaLevel` that will set the [aria-level](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-level) of this div instead?

Reply:
Thanks @jsnmoon I've made some changes - is this kind of what you had in mind?

Reply:
Looks good!